packages/components/src/__future__/Select/Select.spec.tsx

```diff
@@ -315,13 +315,14 @@ describe("<Select />", () => {
       })
 
       it("selects the option when hits enter on a non-selected option", async () => {
-        const { getByRole } = render(<SelectWrapper defaultOpen />)
+        const { getByRole } = render(<SelectWrapper />)
 
         await user.tab()
+        await user.keyboard("{ArrowDown}")
         await waitFor(() => {
           expect(
             getByRole("option", { name: "Short black", selected: false })
-          ).toHaveFocus()
+          ).toBeVisible()
```

Comment:
Options don't get real focus in this component. It was only getting focus here because of the focus lock that was added in the migration PR.

No idea why, but the `defaultOpen` was causing these tests to no longer work. It should have been as simple as reverted back to what kaizen-legacy is doing with these tests but something else must have changed in the migration.

Reply:
The final selection verifies that the correct thing was "focused" no?

I assumed this was just in here so that the test runner didn't press enter before the listbox is visible.